.github/workflows/iroha2-dev-pr-wasm.yaml

```diff
@@ -23,23 +22,6 @@ env:
   RUSTUP_TOOLCHAIN: nightly-2023-06-25
 
 jobs:
-  static-analysis:
-    runs-on: ubuntu-latest
-    container:
-      image: hyperledger/iroha2-ci:nightly-2023-06-25
-    steps:
-      - uses: actions/checkout@v3
-      - uses: Swatinem/rust-cache@v2
-
-      - name: Format
-        run: cargo fmt --all -- --check
-      - name: Lints
-        if: always()
-        run: cargo clippy -Zlints --workspace --benches --tests --examples --quiet
-      - name: Documentation
-        if: always()
-        run: cargo doc --no-deps --quiet
-
```

Comment:
Why do we choose to stop checking lints for the wasm related code here?

Reply:
we don't, it's part of the workspace now, it should be linted in `iroha2-dev-pr-static.yml`